<launch>
  <node name="tbf_gripper_autonomy_controller" pkg="tbf_gripper_autonomy" type="arm.py" respawn="false" output="screen">
    <rosparam command="load" file="$(find julius_moveit)/config/kinematics.yaml" ns="kinematics"/>
    <rosparam command="load" file="$(find tbf_gripper_autonomy)/config/arm_params.yaml"/>
  </node>
</launch>
